src/pages/ConfirmationPage.js

```diff
@@ -70,6 +70,17 @@ class ConfirmationPage extends React.Component {
     }
   }
 
+  hashFromData( email, paperFileNumber ){
```

Comment:
Where did this algorithm come from? 

Can you guarantee that there will be no collisions on this data? 

What is it going to be used for is it unique for the whole system or just per location?

Why is this being generated on the client side and not in the API when the data gets saved to the database.

Reply:
Sounds good.